client/components/App.jsx

```diff
@@ -30,29 +30,37 @@ const Button = styled.button`
 `;
 
 const CustomerQuestions = () => {
-  const [customerQuestionsData, setCustomerQuestionsData] = useState(null);
+  const [customerQuestionsData, setCustomerQuestionsData] = useState();
   const [questionAnswers, setQuestionAnswers] = useState(customerQuestionsData);
   const [showAmt, setShowAmt] = useState(3);
   const [searchResults, setSearchResults] = useState([]);
   const [isSearching, setIsSearching] = useState(false);
+  const [httpStatusCode, setHttpStatusCode] = useState();
   let dataToShow;
 
-  if (customerQuestionsData !== null) {
+  if (customerQuestionsData) {
     dataToShow = customerQuestionsData.slice(0, showAmt);
   }
 
   const handleSearch = (searchTerm) => {
-    console.log(`searching for ${searchTerm}`)
+    if (searchTerm === '') {
+      setIsSearching(false);
+    } else {
+      console.log(`searching for ${searchTerm}`)
+      setIsSearching(true);
+    }
   }
 
 
   const getCustomerQuestionsData = (productId) => {
     axios.get(`/customer-questions/${productId}`)
```

Comment:
Have you tested if this get request will work for both your local server and proxy server? Do you not need to add http://localhost/4001 before?

Reply:
This service still WIP. I have yet to test with proxy/deployment, but will definitely be adding this. Thank you!